indico/modules/categories/templates/management/_events_list.html

```diff
@@ -0,0 +1,150 @@
+                                        desc=(0 if direction == 'desc' and order_column == 'start_dt' else 1),
+                                        order='start_dt') }}"
+                       class="{{ direction if order_column == 'start_dt' else ''}}">{% trans %}Start date{% endtrans %}</a>
+                </th>
+                <th class="i-table actions"></th>
+            </tr>
+        </thead>
+        {% for event in events.items %}
+            {{ event_row(event) }}
+        {% endfor %}
+    </table>
+    {% if events.pages > 1 %}
+        {% call(page) render_pagination(events) %}
+            {{ url_for('.manage_content', category, page=page,
+                       desc=(1 if direction == 'desc' else 0), order=order_column) }}
+        {% endcall %}
+    {% endif %}
+{% endmacro %}
+
+{% macro render_events_list(category, page, order_column, direction, events) %}
+    <div class="toolbar event-management-toolbar">
+        <div class="group">
+            <a class="i-button icon-checkbox-checked arrow js-dropdown" data-toggle="dropdown"></a>
+            <ul class="dropdown">
+                <li>
+                    <a href="#" data-select-all="#event-management input:checkbox">
+                        {% trans %}All{% endtrans %}
+                    </a>
+                </li>
+                <li>
+                    <a href="#" data-select-none="#event-management input:checkbox">
+                        {% trans %}None{% endtrans %}
+                    </a>
+                </li>
+            </ul>
+        </div>
+        <div class="group">
+            <a href="#" class="i-button js-dropdown arrow" data-toggle="dropdown">
+                {% trans %}Create event{% endtrans %}
+            </a>
+            <ul class="dropdown">
+                <li>
+                    <a href="{{ url_for('event_creation.conferenceCreation', categId=category.id, event_type='lecture') }}">
+                        {% trans %}Create lecture{% endtrans %}
+                    </a>
+                </li>
+                <li>
+                    <a href="{{ url_for('event_creation.conferenceCreation', categId=category.id, event_type='meeting') }}">
+                        {% trans %}Create meeting{% endtrans %}
+                    </a>
+                </li>
+                <li>
+                    <a href="{{ url_for('event_creation.conferenceCreation', categId=category.id, event_type='conference') }}">
+                        {% trans %}Create conference{% endtrans %}
+                    </a>
+                </li>
+            </ul>
+        </div>
+        {% if events.total %}
+            {% if page == 'all' %}
+                <div class="group">
+                    <a href="{{ url_for('.manage_content', category, page=1, order=order_column,
+                                        desc=(1 if direction == 'desc' else 0)) }}"
+                       class="i-button">{% trans %}Paginate the results{% endtrans %}</a>
```

Comment:
move `{% trans %}Paginate the results{% endtrans %}` into its own line (more readable)